_content/team.json

```
@@ -39,6 +39,14 @@
     "lastName": "Ludemann",
     "title": "Principal Software Engineer"
   },
+  {
+    "active": true,
+    "id": "zchay-dolan",
+    "image": "img/team/zchay-dolan.jpg",
+    "name": "Zachary Chay-Dolan",
+    "lastName": "Chay-Dolan",
+    "title": "Software Engineer"
```

We've only [very recently](https://github.com/b2io/base2.io/commit/4ec94670da52e33c52092809d893c2f416dce609) changed from having everyone labeled as "Developer" to having role titles publicly listed. So given the caveat that we're definitely still trying to sort through how to use those public definitions most effectively, I think it makes the most sense to update your title to "Associated Software Engineer".

I'll add a note to the weekly agenda for Thursday to keep the discussion going about how to best position those public roles.

```suggestion
    "title": "Associate Software Engineer"
```